packages/terra-action-footer/src/terra-dev-site/doc/action-footer/Block.c.doc.mdx

```diff
@@ -0,0 +1,27 @@
+import { Badge } from '../../../../package.json?dev-site-package';
```

Comment:
Personal preference, but as we uplift these documents, I think we should number the sort ordering. Instead of alphabetizing it. 

Block.c.doc.mdx -> Block.3.doc.mdx

Reply:
terra-clinical does that. Way easier to follow